tests/clients/test_dapr_grpc_client.py

```diff
@@ -23,17 +23,24 @@
 from dapr.clients.grpc.client import DaprGrpcClient
 from dapr.clients import DaprClient
 from dapr.proto import common_v1
+from tests.clients.fake_client import FakeDaprClient
 from .fake_dapr_server import FakeDaprSidecar
 from dapr.conf import settings
 from dapr.clients.grpc._helpers import to_bytes
 from dapr.clients.grpc._request import TransactionalStateOperation
 from dapr.clients.grpc._state import StateOptions, Consistency, Concurrency, StateItem
+from dapr.clients.grpc._response import (
+    ConfigurationItem
+)
+
+from tests.actor.utils import _run
```

Comment:
I know this is convenient, but please don't use that here because it's just strange to import from the actor specific test utilities.